packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -445,6 +448,132 @@ describe('plugin-meetings', () => {
           });
         });
       });
+      describe.only('BNR', () => {
+        describe('#enableBNR', () => {
+          it('should have #enableBnr', () => {
+            assert.exists(meeting.enableBNR);
+          });
+
+          describe('before audio attached to meeting', () => {
+            it('should throw no audio error', async () => {
+              await meeting.enableBNR().catch((err) => {
+                assert.equal(err.toString(),'Error: Meeting doesn\'t have an audioTrack attached');
+              });
+            });
+          });
+
+          describe('after audio attached to meeting', () => {
+            const fakeMediaTrack = () => ({stop: () => {}, readyState: 'live', getSettings: () => {
+              return {
+                'frameRate': 48000
+              }
+            }});
+
+            class FakeMediaStream {
+
```

Comment:
formatting issues.

Reply:
This is because we have linting error in husky and we've force pushed, once we have linter up and running, we will lint once and push those files for commit.